lib/apipie/version.rb

```diff
@@ -1,3 +1,3 @@
 module Apipie
-  VERSION = '0.5.18'
+  VERSION = '0.5.19'
```

Comment:
I wasn't certain if I should bump this up or not. Can revert back to `0.5.18` if that is the preferred way.

Reply:
We're just using a commit ref for Apipie right now so version number isn't too important.

We might want to leave the version as-is for now to prevent confusion when the upstream version updates to 0.5.19. I don't really have a strong opinion either way though :P